syntax = "proto3";

enum PBToolType {
  NONE_TT = 0;
  ROLLING_BLADE_TT = 1;
  DRAG_KNIFE_TT = 2;
  TANGENTIAL_BLADE_TT = 3;
  PEN_DT_TT = 4;
  STYLUS_TT = 5;
  SCORING_WHEEL_TT = 6;
  PERFORATION_WHEEL_TT = 9;
  DEBOSSING_TOOL_TT = 10;
  WAVE_TOOL_TT = 11;
  ENGRAVE_TOOL_TT = 12;
  MIDAS_FINE_TOOL_TT = 13;
  MIDAS_REGULAR_TOOL_TT = 14;
  MIDAS_BOLD_TOOL_TT = 15;
  DOUBLE_SCORING_WHEEL_TT = 98;
  DEEP_CUT_TT = 99;
}

enum PBArtType {
  NONE_ART_TYPE = 0;
  CUT_ART_TYPE = 1;
  DRAW_ART_TYPE = 2;
  SCORE_ART_TYPE = 4;
  DEBOSS_ART_TYPE = 5;
  WAVE_ART_TYPE = 6;
  ENGRAVE_ART_TYPE = 7;
  PERFORATE_ART_TYPE = 8;
  MIDAS_FINE_ART_TYPE = 10;
  MIDAS_REGULAR_ART_TYPE = 11;
  MIDAS_BOLD_ART_TYPE = 12;
}
